go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/Microsoft/hcsshim v0.8.9 // indirect
-	github.com/blang/semver v3.5.1+incompatible
+	github.com/blang/semver/v4 v4.0.0
```

Comment:
Why was this necessary?

Reply:
The `Version` in `ClusterServiceVersionSpec` requires semver/v4. It didn't make sense to keep both semver versions around, and since operator-lifecycle-manager already uses v4 it seemed like a good choice to keep.

Reply:
Could you link where exactly you need `semver/v4` in the `ClusterServiceVersionSpec` for this PR?

Reply:
https://github.com/operator-framework/operator-registry/pull/826/files#diff-bc4c211037518ff105dee0994c7e5fe4ae00bfd9e3230cb17b82605894dfc33aR3083

That requires `OperatorVersion`, which is `semver/v4`

Reply:
Okay that makes sense.